cou/steps/execute.py

```diff
@@ -57,68 +57,86 @@ def normal(text: str) -> str:
         """
         return Style.RESET_ALL + text + Style.RESET_ALL
 
-    return normal(parameter + "Continue(") + bold("y") + normal("/") + bold("n") + normal("):")
+    return normal(parameter + "\nContinue (") + bold("y") + normal("/") + bold("n") + normal("): ")
```

Comment:
Personally I don't feel like it is necessary because the colored check mark indicates it is part of a previous step. But if you think it would look better having a new line added in-between, I can add it.

Reply:
It's really no big deal and no blocker, but I think the display is better with an empty line between applications upgrade.

Reply:
Added!